src/main/java/seedu/address/logic/commands/CategorizeTagCommand.java

```diff
@@ -51,6 +51,7 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(String.format(MESSAGE_DUPLICATE_CATEGORY, targetTag, updatedCategory));
         }
         model.setTagsCategory(targetTag, updatedCategory);
+        model.refreshCampusConnect();
```

Comment:
This would most definitely refresh the PersonList whenever the command is executed.